packages/main/src/DateRangePicker.js

```diff
@@ -211,23 +211,25 @@ class DateRangePicker extends DatePicker {
 		}
 		this.valueState = ValueState.None;
 
-		const firstDate = this.getFormat().parse(dates[0]);
-		const secondDate = this.getFormat().parse(dates[1]);
+		let firstDate = this.getFormat().parse(dates[0]);
```

Comment:
if firstDate does not change, it should be "const" by convention

Reply:
firstDate variable value could be changed later on if the firstDate and lastDate variable values need to be switched.